src/otx/v2/adapters/torch/mmengine/mmseg/modules/models/utils/psp_layer.py

```diff
@@ -0,0 +1,51 @@
+"""PSP module."""
+# Copyright (c) 2019 MendelXu
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+from __future__ import annotations
+
+from typing import ClassVar
+
+import torch
+from torch import nn
+
+
+class PSPModule(nn.Module):
```

Comment:
Is PSP a well-known module in the recent literature? If not would it be an idea to make the naming a bit more explicit?

Reply:
I renamed it to `PyramidScenePooling`